### DISCUSSION TITLE
Deployment to Azure App Service

### DISCUSSION THREAD

**Discussion by fleed** [payloadcms/payload#1200] [Q&A]
Hello

I'm trying to deploy Payload CMS to an Azure App Service application.

I created a GitHub workflow where I run the `npm install` and `npm run build` commands, then I upload the content of the `build` directory to Azure.
Anyway I think that some required files (e.g. `server.js`) are missing.
I couldn't find any specific docs covering this scenario.

Are there any resources that I can use for reference?

Thank you

**Comment by slavanga** [comment]
@DanRibbens @jmikrut could you give us some guidance for deploying Payload CMS to Azure?
